packages/manager/modules/carrier-sip/src/cdr/translations/Messages_fr_FR.json

```
@@ -0,0 +1,11 @@
+{
+  "carrier_sip_cdr_title": "CDR",
+  "carrier_sip_cdr_information": "Le Call Details Records (CDR) contient toutes les informations relatives à un appel téléphonique : numéro de téléphone, durée d’appel, date et heure de l’appel, etc. Obtenez le détail des appels passés en téléchargeant le fichier .csv après avoir sélectionné la période souhaitée (mois et année) ci-dessous.",
```

```suggestion
  "carrier_sip_cdr_information": "Le Call Details Records (CDR) contient toutes les informations relatives à un appel téléphonique : numéro de téléphone, durée d’appel, date et heure de l’appel, etc. Obtenez le détail des appels passés en téléchargeant le fichier .csv après avoir sélectionné la période souhaitée (mois et année) ci-dessous.",
```
insecable space 😅  don't worry about it